website/blog/2023-08-18-ferretdb-going-postgres-ibiza-2023.md

```diff
@@ -0,0 +1,25 @@
+---
+slug: ferretdb-going-postgres-ibiza-2023
+title: 'FerretDB is going to Postgres Ibiza 2023'
+author: Peter Benei
+description: >
+  There would be no FerretDB with Postgres; Postgres Ibiza is an event that we wouldn’t miss this year. Let’s meet and spend some amazing time together!
+image: /img/blog/postgres-ibiza-2023.jpg
+tags: [events, community, open source]
+---
+
+![FerretDB is going to Postgres Ibiza 2023](/img/blog/postgres-ibiza-2023.jpg)
+
+**There would be no [FerretDB](https://www.ferretdb.io) with [Postgres](https://www.postgresql.org/). We heavily rely on not just the software, but also its community to make FerretDB happen. It is why Postgres Ibiza is an event that we wouldn't miss this year. Let's meet and spend some amazing time together!**
```

Comment:
I think the intended statement here is that there would be no FerretDB withOUT Postgres